src/main/java/org/ethereum/beacon/discovery/schema/EnrField.java

```diff
@@ -4,20 +4,65 @@
 
 package org.ethereum.beacon.discovery.schema;
 
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
 /** Fields of Ethereum Node Record */
```

Comment:
Might be worth leaving a pointer to the docs here: https://github.com/ethereum/devp2p/blob/master/enr.md